src/components/VSearchRating.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div class="caption font-semibold">
+  <div>
```

Comment:
The exported name of this component needs the `V` too.

Reply:
Great catch!